packages/parse/src/main.js

```diff
@@ -1 +1,22 @@
-export { default as bootPeg } from './boot-peg.js';
+import { bootPeg, bootPegAst, makePeg, makeJSON, makeJustin } from './all.js';
+
+export { bootPeg };
+
+export { flattenParserTag } from './all.js';
+
+// The currently-supported parse tags.
+
+/**
+ * Parser template tag that creates new parser template tags from PEGs.
+ */
+export const peg = bootPeg(makePeg, bootPegAst);
+
+/**
+ * JSON parser template tag.
+ */
+export const json = makeJSON(peg);
+
+/**
+ * Justin parser template tag.
+ */
+export const justin = makeJustin(peg.extends(json));
```

Comment:
ok... I got this to work in the case of:

```js
justin`make(BLD, BigInt(100))`
```

My first attempt was:

```js
justin`make(BLD, 100n)`
```

The diagnostic was pretty misleading: `quasiRest is not a function`.

Reply:
I fixed this error.  Now it returns the diagnostic I was expecting:

```
justin`make(BLD, 100n)`
-------template--------
"make(BLD, 100n)"

-------
looking for digit, frac, exp, _WSN
Syntax error at 13 "n" #0:13
```

and I added support for bigint literals, so now you should get:

```
[
  'call',
  [ 'use', 'make', _pegPosition: '"m" #0:0' ],
  [
    [ 'use', 'BLD', _pegPosition: '"B" #0:5' ],
    [ 'data', 100n, _pegPosition: '"1" #0:10' ],
    _pegPosition: '"(" #0:4'
  ],
  _pegPosition: '"m" #0:0'
]
```